### PR TITLE
add rencfs project

### PR DESCRIPTION
**Pull request by radumarias**

### CODE REVIEW

File: _data/projects/rencfs.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: rencfs
+desc: An encrypted file system in Rust that mounts with FUSE on Linux. It can be used to create encrypted directories.
+site: https://github.com/radumarias/rencfs
+tags:
+  - v0.13.56
```
Comment (shiftkey) [inline _data/projects/rencfs.yml]: 🤔 this should be a descriptive tag, rather than a version tag

File: _data/projects/rencfs.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+upforgrabs:
+  name: good first issue
+  link: https://github.com/radumarias/rencfs/issues?q=is%3Aopen+is%3Aissue+label%3Afirst-timers-only+label%3A%22good+first+issue%22
```
Comment (shiftkey) [inline _data/projects/rencfs.yml]: We generally advise to use one label here, instead of filtering on multiple...

**Comment by radumarias** [comment]
changed based on review

**Review by shiftkey — APPROVED** [review]

**Comment by radumarias** [comment]
thank you

**Pull request merged**